api/src/Feature.FormTemplates/AssignTemplates/Endpoint.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Feature.FormTemplates.Specifications;
+using Microsoft.EntityFrameworkCore;
+using Vote.Monitor.Domain;
+using Vote.Monitor.Domain.Entities.ElectionRoundAggregate;
+using Vote.Monitor.Domain.Entities.ElectionRoundFormTemplateAggregate;
+
+namespace Feature.FormTemplates.AssignTemplates;
+
+public class Endpoint(
+    IReadRepository<ElectionRound> electionRoundRepository,
+    IReadRepository<FormTemplateAggregate> formTemplateRepository,
+    IRepository<ElectionRoundFormTemplate> electionRoundFormTemplateRepository
+    ) :
+        Endpoint<Request, Results<NoContent, NotFound>>
+{
+    public override void Configure()
+    {
+        Post("/api/election-rounds/{electionRoundId}/form-templates:assign-templates");
+        Policies(PolicyNames.PlatformAdminsOnly);
+    }
+
+    public override async Task<Results<NoContent, NotFound>> ExecuteAsync(Request req, CancellationToken ct)
+    {
+        var electionRound = await electionRoundRepository.GetByIdAsync(req.ElectionRoundId,ct);
```

Comment:
this line will fetch an election round from db. A better way to check if it exists is to use `AnyAsync` with a specification for selecting election round by id